.eslintrc.js

```
@@ -27,6 +27,9 @@ module.exports = {
     '@typescript-eslint/camelcase': 'off',
     '@typescript-eslint/ban-ts-comment': 'off',
     '@typescript-eslint/explicit-module-boundary-types': 'off',
+    '@typescript-eslint/no-unused-vars': ['warn', {
```

ESLint

[error] Replace `'warn',` with `⏎······'warn',⏎·····`

(prettier/prettier)